src/Dapps/store.js

```diff
@@ -123,6 +124,17 @@ export default class DappsStore extends EventEmitter {
 
         return this.fetchRegistryApp(dappReg, id, true);
       })
+      .then((app) => {
+        if (app.type === 'network') {
+          return HashFetch.get().fetch(this._api, app.contentHash, 'dapp')
+            .then(appPath => {
+              app.localUrl = `file://${appPath}/index.html`;
```

Comment:
Does it work on windows? I thought that `///` is required

Reply:
Good question! Need to check